.pre-commit-hooks.yaml

```diff
@@ -183,3 +183,13 @@
     entry: trailing-whitespace-fixer
     language: python
     types: [text]
+-  id: update-commit-msg
+   name: update-commit-msg
+   entry: update-commit-msg
+   # Update to your ticket matching regex
+   args: ['--regex=TICKET-[0-9]']
+   language: python
+   stages: [commit-msg]
+   additional_dependencies: ['six', 'invoke']
```

Comment:
these should be specified in setup.py, six is already there, invoke is (imo) unnecessary because you can just use `subprocess.check_output`